dvc/data/stage.py

```diff
@@ -168,7 +168,7 @@ def _build_tree(fs_path, fs, name, **kwargs):
         assert key
         tree.add(key, meta, obj.hash_info)
 
-        tree_meta.size += meta.size
+        tree_meta.size += meta.size or 0
```

Comment:
Excuse me, why can not just set them default 0? Like in `C` or `Java` current `None` is just like `null` point in other languages.